storage/protocol_state.go

```diff
@@ -5,18 +5,19 @@ import (
 	"github.com/onflow/flow-go/storage/badger/transaction"
 )
 
-// ProtocolState represents persistent storage for protocol state entries.
-type ProtocolState interface {
+// EpochProtocolStateEntries represents persistent storage for epoch protocol state entries.
+// Epoch Protocol State entries hold fork-aware sub-states of the overall Protocol State (KV Store).
+type EpochProtocolStateEntries interface {
 	// StoreTx returns an anonymous function (intended to be executed as part of a badger transaction),
 	// which persists the given protocol state as part of a DB tx. Per convention, the identities in
 	// the Protocol State must be in canonical order for the current and next epoch (if present),
 	// otherwise an exception is returned.
 	// Expected errors of the returned anonymous function:
 	//   - storage.ErrAlreadyExists if a Protocol State with the given id is already stored
-	StoreTx(protocolStateID flow.Identifier, protocolState *flow.ProtocolStateEntry) func(*transaction.Tx) error
+	StoreTx(protocolStateID flow.Identifier, protocolState *flow.EpochProtocolStateEntry) func(*transaction.Tx) error
```

Comment:
To reduce ambiguity, I would rename `protocolStateID` to `epochProtocolStateID` for **all respective method inputs in this file**. Otherwise, people might erroneously assume that this would be the ID of the entire protocol state (entire KV store) as included in the block [`Paylod`](https://github.com/onflow/flow-go/blob/0dd09d63678dfd073b7f658480ba92105031eb32/model/flow/payload.go#L8-L24): https://github.com/onflow/flow-go/blob/0dd09d63678dfd073b7f658480ba92105031eb32/model/flow/payload.go#L23

Reply:
Done in https://github.com/onflow/flow-go/pull/5968/commits/4600168640d01c96779d705a525fb0f51888f182